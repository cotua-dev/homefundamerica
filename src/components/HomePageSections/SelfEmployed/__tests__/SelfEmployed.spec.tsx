import React from 'react';
import { render, RenderResult } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import { default as SelfEmployed } from '..';

describe('SelfEmployed component', (): void => {
    test('renders', (): void => {
        const { container }: RenderResult = render(<SelfEmployed/>);

        expect(container.firstChild).toMatchSnapshot();
    });
});
