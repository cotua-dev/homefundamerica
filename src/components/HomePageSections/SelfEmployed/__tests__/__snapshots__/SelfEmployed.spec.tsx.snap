// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SelfEmployed component renders 1`] = `
<section>
  <div>
    <h1>
      Are you self-employed
    </h1>
    <h1>
      and need a mortgage?
    </h1>
    <h1>
      We can Help!
    </h1>
    <h3>
      Self-employed borrowers looking for a mortgage can qualify without tax returns
    </h3>
    <a
      href="/conventional"
    >
      <button
        type="button"
      >
        Apply Now
      </button>
    </a>
    <a
      href="tel:8888186393"
    >
      (888) 818-6393
    </a>
  </div>
</section>
`;
